### ISSUE TITLE
False positive operation and fragment duplication due to old version of graphql-tools/load

### ISSUE THREAD

**Issue by tobiasdiez** [kamilkisiela/graphql-inspector#1991]
**Describe the bug**

Code such as 
```ts
gql(/* GraphQL */ `
        mutation Logout {
          logout
        }
      `
export const DocumentForView = gql(/* GraphQL */ `
  fragment DocumentForView on Document {
    id
    title
    keywords
    abstract
    authors
`)

```
produced false errors 
```
 - There can be only one operation named Logout.
 - Name of DocumentForView fragment is not unique
```

I traced it down to 
https://github.com/kamilkisiela/graphql-inspector/blob/b7f6e1cd04a01fbbd4c7b73ca181f522402e0a01/packages/graphql-cli/validate/src/index.ts#L105
where already the loaded documents have a rawSDL that looks similar to the following:
```
"fragment DocumentForView on Document {\n  id\n  title\n  keywords\n  abstract\n  authors {\n    ... on Person {\n      id\n      name\n    }\n    ... on Organization {\n      id\n      name\n    }\n  }\n  ... on JournalArticle {\n    in {\n      journal {\n        id\n        name\n      }\n    }\n  }\n  ... on ProceedingsArticle {\n    in {\n      title\n    }\n  }\n  ... on Thesis {\n    institution {\n      id\n      name\n    }\n  }\n}\n\nfragment DocumentForView on Document {\n  id\n  title\n  keywords\n  abstract\n  authors {\n    ... on Person {\n      id\n      name\n    }\n    ... on Organization {\n      id\n      name\n    }\n  }\n  ... on JournalArticle {\n    in {\n      journal {\n        id\n        name\n      }\n    }\n  }\n  ... on ProceedingsArticle {\n    in {\n      title\n    }\n  }\n  ... on Thesis {\n    institution {\n      id\n      name\n    }\n  }\n}"
```
As you can see, the fragment is repeated twice.
Thus, I guess this is not really a problem with graphql-inspector itself but more with graphql-tools/load. Thus I would suggest to fix this problem by updating graphql-tools/load from 6.0.0 to the most recent version 7.x.

**To Reproduce**
Run `graphql-inspector validate` against the above graphql document.
See https://github.com/JabRef/JabRefOnline/pull/356 for a full example.

**Expected behavior**
No error, as fragment and operation is only defined a single time.

**Environment:**

- OS:
- `@graphql-inspector/...`: 2.7
- `graphql`: ?
- NodeJS: 14

**Comment by kamilkisiela** [comment]
released in v3.0.0

**Comment by tobiasdiez** [comment]
Thanks @kamilkisiela! It is is indeed fixed now. 🚀
